composer.json

```diff
@@ -29,7 +29,8 @@
     },
     "require-dev": {
 
-        "phing/phing": "2.14.0"
+        "phing/phing": "2.14.0",
```

Comment:
awesome. will try that and see what happens. if works, then can simplify instructions.
(this will also be very helpful since also use phing to clean up the public/assets/ stuff)